core-kotlin/src/test/java/com/amplifyframework/kotlin/auth/KotlinAuthFacadeTest.kt

```diff
@@ -870,7 +872,7 @@ class KotlinAuthFacadeTest {
      */
     @Test(expected = AuthException::class)
     fun getCurrentUserWhenUserNameIsEmpty(): Unit = runBlocking {
-        val expectedException = AuthException.InvalidUserPoolConfigurationException()
+        val expectedException = AuthException("uh", "oh")
```

Comment:
can we fix this messaging?

Reply:
This is just a test where an expected exception is passed. This isn't in production.